emData/download.sh

```diff
@@ -4,6 +4,11 @@
 memprints_url="https://cernbox.cern.ch/index.php/s/y7IWeDG4x7Sg7Im/download"
 luts_url="https://cernbox.cern.ch/index.php/s/DuhCjcykSHZLRhM/download"
 
+# Supplemental test-vectors for development of the PurgeDuplicates module.
```

Comment:
Please update comment, since the PR it refers to to emulation has already been merged.

Reply:
Done.